brave/brave-core
# Implement "Desktop Welcome Pages" feature (v2)

Resolves https://github.com/brave/brave-browser/issues/40157
Resolves https://github.com/brave/brave-browser/issues/42562

(for Brave employees)
Privacy / Security review
- v1 https://github.com/brave/reviews/issues/1705
- v2 https://github.com/brave/reviews/issues/1758
- v3 https://github.com/brave/reviews/issues/1848



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Visit `brave://getting-started`
- Verify the page loads correctly and that all "browser action" buttons on the page work as expected.
- Enable the flag `#brave-show-getting-started-page` and restart the browser.
- Visit `brave://welcome`
- Navigate through the welcome UI and click the "Finish" button.
- Verify that `brave://getting-started` is loaded correctly.
